src/pybind/mgr/orchestrator/module.py

```diff
@@ -808,6 +808,7 @@ def _list_daemons(self,
                       service_name: Optional[str] = None,
                       daemon_type: Optional[str] = None,
                       daemon_id: Optional[str] = None,
+                      group_by: Optional[str] = 'name',
```

Comment:
```suggestion
                      sort: Optional[str] = 'name',
```

similar to `podman ps --sort names` ?

Reply:
@mgfritch I used `sort_by` as only `sort` to me sounds more generic.